api-metastore/src/main/java/org/zalando/nakadi/model/AuthorizationAttributeQueryParser.java

```diff
@@ -1,6 +1,6 @@
 package org.zalando.nakadi.model;
 
-import org.apache.commons.lang.StringUtils;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
Is it a compile- or a run-time error if we don't change this?

Reply:
I notice we are already using a mix of the two (in different sub-projects):
```
import org.apache.commons.lang.StringUtils;
import org.apache.commons.lang3.StringUtils;
```

That's not good.  We should fix it, but I'd prefer to do that separately from bumping ZooKeeper version.

Reply:
Ah, I see now that these are the only two uses of `lang` that are left there. OK.

Reply:
That would be compile error, so the first commit is actually per-requisite for the version bump.
The updated Zookeeper client version is not using the old `org.apache.commons` lib anymore. The Nakadi project was using it as transitive dependency.